paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountFormFragment.kt

```diff
@@ -424,7 +426,10 @@ internal class USBankAccountFormFragment : Fragment() {
                                 .height(40.dp)
                                 .width(56.dp)
                         )
-                        Text(text = "$displayName ••••$last4")
+                        Text(
```

Comment:
It feels like in a later refactor this should be the same UI component, something like TextOnComponent?

Also it seems like this should be onBackground or onSurface if this is lying directly on top of the paymentsheet.

Reply:
This is sitting on top of a `SectionCard`

Reply:
ah, I had the wrong one, that makes perfect sense!